sbin/bectl/bectl.8

```diff
@@ -397,6 +397,7 @@ will force the unmount if busy.
 .El
 .Pp
 Unmount from a temporary mount point will not remove the mount point.
```

Comment:
Indenting: how can I align the paragraph at line 399 with the preceding paragraph (lines 393–397)?